tests/homework1/task2/test_wc.py

```diff
@@ -0,0 +1,35 @@
+import os
+import unittest
```

Comment:
Your testing is great, I wonder why you only tested the `wc` function. However, I will not ask you to test all the other ones, because you already did enough of testing :)